packages/core/src/indexing/database-index-api.ts

```diff
@@ -197,6 +207,15 @@ export class DatabaseIndexApi<DateType = Date | number> {
       return StreamID.fromString(result.model)
     })
   }
+  private async getPreviouslyIndexedModelsFromDatabase(): Promise<Array<StreamID>> {
+    return (
+      await this.dbConnection(INDEXED_MODEL_CONFIG_TABLE_NAME).select('model').where({
+        is_indexed: false,
```

Comment:
Shouldn't this be true if we're getting the models that were previously indexed?

Reply:
I was under the impression that the models to prevent re-indexing for had their indexing explicitly stopped and had the _is_indexed_ field set to false based on the ticket description: "Prevent models that are already set in ceramic_models:is_indexed to false and the corresponding MID table exists the DB don't allow to re-enable this model."

Reply:
Previously would be `false` - currently is `true`.